### DISCUSSION TITLE
Education Benefit not received after one week of approval

### DISCUSSION THREAD

**Discussion by icyice69** [community/community#131147] [GitHub Education]
### Hi everyone,

could you please help me with getting GitHub Co-Pilot to work? My application for the GitHub Education Benefits has been approved over a week ago but I am still not able to use it as it says I do not have access to it in my IDE.

Thanks in advance!

**Comment by abhishekjaiswar221** [comment]
### @icyice69 Hello bro,
- You can find out what your current plan is by going to https://github.com/settings/billing/summary. If you have Github Pro as your current subscription, your student developer pack is active. 
- After that, you can check down for the Add-Ons section and find github copilot over there, and then simply enable that for your account and head to your IDE.
#### 1: Visual Studio Code: Go to https://marketplace.visualstudio.com/items?itemName=GitHub.copilot to download this extension.
#### 2: JetBrains IDEs: Visit this link and download this plugin: https://plugins.jetbrains.com/plugin/17718-github-copilot. 
#### After that, sign in to your IDEs using your github account which has your student pack enabled, and your github copilot plugin will instantly be activated.

**Comment by emansarahafi** [comment]
When you have the GitHub Pro badge on your profile, you can use the benefits. But sometimes, it takes a longer time to show for other users. I recommend you contact someone from the GitHub Education staff who could fix it for you by opening a ticket [here](https://support.github.com/contact/account), selecting Other, and explaining your situation.

**Comment by abhishekjaiswar221** [comment]
#### @emansarahafi  Some times the PRO badge gets disabled form https://github.com/settings/profile (scroll down to Profile Settings) and does not show up in your profile overview

**Reply by emansarahafi** [reply]
Yes, I am just saying so supposing that he did not disable the badge from showing on his profile.
